docker/kyuubi-pod.yaml

```diff
@@ -17,20 +17,25 @@
 
 # Usage:
 # Start:
-#   kubectl apply -f https://raw.githubusercontent.com/apache/incubator-kyuubi/master/docker/example.yaml
-# Connect:
+#   kubectl apply -f https://raw.githubusercontent.com/apache/incubator-kyuubi/master/docker/kyuubi-pod.yaml
+# Expose port:
+# 1. for tmp test
 #   kubectl port-forward kyuubi-example --address localhost 10009:10009
-# Or
-#   kubectl exec -it kyuubi-example -- /bin/bash
+#   Connect:
+#     kubectl exec -it kyuubi-example -- /bin/bash
 #
-#   ${SPARK_HOME}/bin/beeline -u 'jdbc:hive2://localhost:10009'
+#     ${SPARK_HOME}/bin/beeline -u 'jdbc:hive2://localhost:10009'
+# 2. use NodePort
```

Comment:
Expose the Pod as a service with NodePort

Reply:
Had modify.